crates/miden-tx/src/executor/data_store.rs

```diff
@@ -31,7 +31,7 @@ pub trait DataStore: MastForestStore {
         &self,
         account_id: AccountId,
         ref_blocks: BTreeSet<BlockNumber>,
-    ) -> impl FutureMaybeSend<Result<(PartialAccount, BlockHeader, PartialBlockchain), DataStoreError>>;
```

Comment:
I wonder if we should change the docs for `DataStore::get_transaction_inputs` to start:
> Returns the data required to prepare the execution of a transaction against the account with the ...

Because "all the data" sounds incorrect since that would actually be `TransactionExecutionInputs`.